CMakeLists.txt

```diff
@@ -1,6 +1,29 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(VkFFT)
 
+include(FetchContent)
+set(_itk_build_testing ${BUILD_TESTING})
+#set(BUILD_TESTING OFF)
+set(_itk_build_shared ${BUILD_SHARED_LIBS})
+set(BUILD_SHARED_LIBS OFF)
```

Comment:
set `VTKFFT_BACKEND` to `3` for OpenCL

https://github.com/DTolm/VkFFT/blob/4608dd00899e3c50cd3cfb3f4b8ef056f09261a9/CMakeLists.txt#L8